sdks/wasm/framework/src/System.Net.Http.WebAssemblyHttpHandler/WebAssemblyHttpHandler.cs

```diff
@@ -1,48 +1,70 @@
-using System;
 using System.IO;
-using System.Net;
-using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using WebAssembly;
 using WebAssembly.Core;
 using WebAssembly.Host;
 
-namespace WebAssembly.Net.Http.HttpClient {
-	public class WasmHttpMessageHandler : HttpMessageHandler {
+namespace System.Net.Http
+{
+	/// <summary>
+	/// <see cref="WebAssemblyHttpHandler" /> is a specialty message handler based on the
+	/// Fetch API for use in WebAssembly environments.
+	/// </summary>
+	/// <remarks>See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API</remarks>
+	public class WebAssemblyHttpHandler : HttpMessageHandler {
 		static JSObject fetch;
 		static JSObject window;
 
 		/// <summary>
-		/// Gets or sets the default value of the 'credentials' option on outbound HTTP requests.
-		/// Defaults to <see cref="FetchCredentialsOption.SameOrigin"/>.
+		/// Configures a value for the 'cache' option for the HTTP request.
 		/// </summary>
-		public static FetchCredentialsOption DefaultCredentials { get; set; }
-		    = FetchCredentialsOption.SameOrigin;
+		/// <remarks>
+		/// See https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials
+		/// </remarks>
+		public RequestCache? RequestCache { get; set; }
 
-		public static RequestCache Cache { get; set; }
-		    = RequestCache.Default;
+		/// <summary>
+		/// Configures a value for the 'mode' option for the HTTP request.
+		/// </summary>
+		/// <remarks>
+		/// See https://developer.mozilla.org/en-US/docs/Web/API/Request/mode
+		/// </remarks>
+		public RequestMode? RequestMode { get; set; }
 
-		public static RequestMode Mode { get; set; }
-		    = RequestMode.Cors;
+		/// <summary>
+		/// Configures a value for the 'credentials' option for the HTTP request.
+		/// </summary>
+		/// <remarks>
+		/// See https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials
+		/// </remarks>
+		public RequestCredentials? RequestCredentials { get; set; }
```

Comment:
Instead of assigning defaults, making these nullable makes it a better approximation of the fetch API where they remain unspecified by default.